### ISSUE TITLE
Missing information on how to flash TSDZ2

### ISSUE THREAD

**Issue by ShalokShalom** [danielnilsson9/bbs-fw#254]
The TSDZ2 have currently no information on how to flash them.

The process on how to wire the TDSZ2 to the flash cable, [is already documented.](https://github.com/OpenSourceEBike/TSDZ2_wiki/wiki/Flash-the-firmware-on-TSDZ2)

Might not be a top priority right now, so I leave this open for further reference. 🙂👍🏻

**Comment by danielnilsson9** [comment]
Yeah, basically that guide works, there is no special configuration when flashing TSDZ2.
